Intersect.Server/Localization/Strings.cs

```diff
@@ -776,6 +776,9 @@ public sealed class ItemsNamespace : LocaleNamespace
             [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
             public readonly LocalizedString stunned = @"You cannot use this item whilst stunned.";
 
+            [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
+            public readonly LocalizedString notyours = @"This item does not belong to you!";
```

Comment:
So the reason all these strings were in lowercase is irrelevant now (according to JC), and they're not really legible. I haven't gone through and fixed everything because I'm lazy, but can you change this new one to `NotYours` and also add the `NotNull` attribute to it?

(`[NotNull, JsonProperty...]` is valid syntax and preferable for this situation instead of `[NotNull] [JsonProperty...]`).